gulpfile.js

```diff
@@ -197,7 +197,7 @@ async function addDistToIndex() {
     console.log(`stderr: ${res.stderr}`);
 }
 
-const cliVersion = '1.14.4';
+const cliVersion = '1.15.1';
```

Comment:
Looks like you're missing the `dist` update.

Reply:
yep, deliberate for the moment, didn't want to pollute the PR with the /dist updates

Reply:
will also need to undo the latest WIP commit that switches in the out folder for the 2 actions